site2/docs/functions-cli.md

```diff
@@ -1,197 +1,83 @@
-tenant | The tenant of a Pulsar Function. |  |
-timeout-ms | The message timeout in milliseconds. |  |
-topics-pattern | The topic pattern to consume from list of topics under a namespace that match the pattern. [--input] and [--topic-pattern] are mutually exclusive. Add SerDe class name for a pattern in --custom-serde-inputs (only supported in Java Function). |  |
-update-auth-data | Whether or not to update the auth data. | false
-user-config | User-defined config key/values. |  |
-window-length-count | The number of messages per window. |  |
-window-length-duration-ms | The time duration of the window in milliseconds. | |
-
-## get
-
-Fetch information about a Pulsar Function.
-
-Name | Description | Default
----|---|---
-fqfn | The Fully Qualified Function Name (FQFN) for the function. | |
-name | The name of a Pulsar Function. |  |
-namespace | The namespace of a Pulsar Function. |  |
-tenant | The tenant of a Pulsar Function. |  |
-
-## restart
-
-Restart function instance.
-
-Name | Description | Default
----|---|---
-fqfn | The Fully Qualified Function Name (FQFN) for the function. | |
-instance-id | The function instanceId (restart all instances if instance-id is not provided. |  |
-name | The name of a Pulsar Function. |  |
-namespace | The namespace of a Pulsar Function. |  |
-tenant | The tenant of a Pulsar Function. |  |
-
-## stop
-
-Stops function instance.
-
-Name | Description | Default
----|---|---
-fqfn | The Fully Qualified Function Name (FQFN) for the function. | |
-instance-id | The function instanceId (restart all instances if instance-id is not provided. |  |
-name | The name of a Pulsar Function. |  |
-namespace | The namespace of a Pulsar Function. |  |
-tenant | The tenant of a Pulsar Function. |  |
-
-## start
-
-Starts a stopped function instance.
-
-Name | Description | Default
----|---|---
-fqfn | The Fully Qualified Function Name (FQFN) for the function. | |
-instance-id | The function instanceId (restart all instances if instance-id is not provided. |  |
-name | The name of a Pulsar Function. |  |
-namespace | The namespace of a Pulsar Function. |  |
-tenant | The tenant of a Pulsar Function. |  |
+## Pulsar admin CLI for functions
+
+The Pulsar admin interface enables you to create and manage Pulsar Functions through CLI. For the latest and complete information, including commands, flags, and descriptions, refer to [Pulsar admin CLI](https://pulsar.apache.org/tools/pulsar-admin/).
+
+
+## YAML configurations for functions
+
+You can configure a function by using a predefined a .yaml file. The following table outlines the required fields and arguments.
+
+| Field Name           | Nested Field                       | Type                       | Related Command Argument   | Description                                                                                                                                                                                                                                      |
```

Comment:
I suggest to put the Nested Field to the last column

Reply:
Nested fields have been moved to sub-tables for a clear view. PTAL.